Tools/CISupport/ews-build/steps.py

```diff
@@ -6564,28 +6565,25 @@ def run(self, BufferLogObserverClass=logobserver.BufferLogObserver):
             defer.returnValue(rc)
             return
 
-        retry_attempts = 3
-        for i in range(1, retry_attempts + 1):
-            self.contributors, errors = yield Contributors.load(use_network=True)
-            for error in errors:
-                self._addToLog('stdio', error)
-            self._addToLog('stdio', '\n')
-            if self.contributors:
-                break
-            if i < retry_attempts:
-                self._addToLog('stdio', f'Retrying, attempt {i + 1} of {retry_attempts}...\n')
+        self.contributors, errors = yield Contributors.load(use_network=True)
+        for error in errors:
+            self._addToLog('stdio', error)
+        self._addToLog('stdio', '\n')
 
         reviewers, log_text = self.extract_reviewers(self.log_observer.getStdout())
         log_text = log_text.rstrip()
         author = self.getProperty('author', '')
 
+        self.summary = ''
         if any(os.path.basename(file).startswith('ChangeLog') for file in self._files()):
             self.summary = 'ChangeLog modified, WebKit only allows commit messages'
             rc = FAILURE
         elif log_text:
             self.summary = log_text.split('\n')[0]  # Display the first error if there are multiple
             rc = FAILURE
-        elif rc == SUCCESS:
+        if 'contributors.json' in self.summary and self.contributors:
```

Comment:
How do we get the text 'contributors.json' inside the summary, is it coming from a previous error message. If so, that doesn't seem like a reliable way. Can we modify the underlying code so that rc is indeed SUCCESS when we are able to load from the disk (instead of us manually setting rc to SUCCESS).

Reply:
Yeah it comes from the error message. Since it's not a shell command, I set it so that `Contributors.load` will reliably return no errors if it succeeds and that will work with the existing logic with the logs.